### PR TITLE
Functionality moved from `core`: ABI, Contract Addresses, Integration Tests

### PR DESCRIPTION
**Pull request by ytham**
- ABI getters
- Contract address getters
- Integration tests migrated from sdk-core (updated w/ ts circuit inputs instead of raw sdk-core inputs)

**NOTE:** This PR has local link pointers; this is fine since the publish-all script will take care of this automatically.

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -24,17 +24,19 @@
     "crypto"
   ],
   "dependencies": {
-    "@axiom-crypto/circuit": "2.0.4",
-    "@axiom-crypto/core": "2.3.5",
+    "@axiom-crypto/circuit": "link:../circuit/js/dist",
```
Comment (jonathanpwang) [inline client/package.json:27]: needs updates after `circuit/` pr

Comment (ytham) [reply client/package.json]: updated

File: client/src/lib/address.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  baseSepolia,
+  mainnet,
+  optimism,
+  optimismSepolia,
+  scroll,
+  scrollSepolia,
+  sepolia
+} from "viem/chains";
+
+const axiomV2QueryAddresses = {
```
Comment (jonathanpwang) [inline client/src/lib/address.ts:14]: add comment about what this is mapping to/from

Comment (ytham) [reply client/src/lib/address.ts:15]: added

**Review by jonathanpwang — APPROVED** [review]
LGTM, I did not carefully check the test cases -- I assume they're modifications of the previous ones.

**Comment by ytham** [comment]
yes, they're all modifications/adaptations of previous ones

**Pull request merged**